modules/exploits/windows/misc/manageengine_eventlog_analyzer_rce.rb

```diff
@@ -0,0 +1,219 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = ExcellentRanking
```

Comment:
@jvazquez-r7 The "-1" is done on purpose, in real life situations there will never be data in negative number large objects. Having said that, of course feel free to change it to ManualRanking :)